pontoon/translate/views.py

```diff
@@ -47,7 +47,7 @@ def static_serve_dev(request, path, insecure=False, **kwargs):
     """
     try:
         # First try to load the file with django's regular serve view.
-        return serve(request, path, insecure=False, **kwargs)
+        return serve(request, path, insecure=True)
```

Comment:
These are just improvements that I found out while reading the documentation, they don't impact anything.